app/assets/stylesheets/hyrax/_hyrax.scss

```diff
@@ -8,7 +8,7 @@
         'hyrax/file_manager', 'hyrax/form-progress', 'hyrax/positioning',
         'hyrax/fixedsticky', 'hyrax/file_upload', 'hyrax/representative-media',
         'hyrax/footer', 'hyrax/select_work_type', 'hyrax/users', 'hyrax/dashboard',
-        'hyrax/sidebar', 'hyrax/controlled_vocabulary';
+        'hyrax/sidebar', 'hyrax/controlled_vocabulary', 'hyrax/work_editor', 'hyrax/accessibility';
```

Comment:
why are you adding `'hyrax/work_editor'` here?  It was removed recently. Maybe there was a rebase problem?